Rakefile

```diff
@@ -9,20 +9,20 @@ Rails.application.load_tasks
 require 'fileutils'
 require 'yaml'
 
-desc 'Quick command to execute the dockerized terminal for running commands'
-task :app_cli do
-  # Run the command ignoring the results
-  sh('docker-compose', '-f', 'docker-compose.yml',
-     '-f', 'docker-compose.test.yml',
-     'run', '--rm', 'app', 'bash') || true
-end
-
-desc 'Quick command to launch the compose stack'
-task :app_run do
-  # Run the command ignoring the results
-  sh('docker-compose', '-f', 'docker-compose.yml', 'up') || true
-end
-
+# desc 'Quick command to execute the dockerized terminal for running commands'
```

Comment:
Can we just remove the commented out lines?

Reply:
Its possible, but I was planning to move the commented lines to a new docker-file.